pgdrive/envs/pgdrive_env.py

```diff
@@ -84,6 +84,7 @@ def default_config() -> PGConfig:
             record_episode=False,
             use_saver=False,
             save_level=0.5,
+            num_player=2,
```

Comment:
I would suggest to create a new environment class, rather than just add on original PGDriveEnv